app/services/data/get-approved-claim-expense-data.js

```diff
@@ -0,0 +1,45 @@
+const config = require('../../../knexfile').asyncworker
+const knex = require('knex')(config)
+
+module.exports.getClaimExpenseData = function(claimId) {
+    // Get all Claim expenses from database for specified claim id
+    return knex('IntSchema.ClaimExpense')
+    .where('IntSchema.ClaimExpense.ClaimId', claimId)
+    .select('IntSchema.ClaimExpense.ClaimExpenseId',
+        'IntSchema.ClaimExpense.ExpenseType',
+        'IntSchema.ClaimExpense.Cost',
+        'IntSchema.ClaimExpense.From',
+        'IntSchema.ClaimExpense.To',
+        'IntSchema.ClaimExpense.IsReturn',
+        'IntSchema.ClaimExpense.ApprovedCost',
+        'IntSchema.ClaimExpense.Status',
+        'IntSchema.ClaimExpense.EligibilityId'
+    )
+}
+
+module.exports.getClaimantData = function(reference, claimId) {
+    var visitor
+    var visitorBankDetails
+
+    return knex('IntSchema.Visitor')
+        .where('Reference', reference)
```

Comment:
you should not go by reference, go by the EligibilityId associated with the claim (join on Claim)